### PR TITLE
rewrote reset function to create all tables.

### PR DESCRIPTION
**Pull request by jefftwin**
reset function now creates all project tables.

### CODE REVIEW

**Comment by maknoon** [comment]
I noticed that the application throws an `InterfaceError: (0, '')` if I try to hit the endpoint in the same db session, which I think is caused by my implementation of a global cursor..... so I think we might have to open and close the cursor each time inside of each db function. if this happens in the future I think that's how we can fix that bug (or just squash it now idk)

**Review by maknoon — APPROVED** [review]

File: airplanedb.py [hunk 1]
```diff
@@ -26,17 +26,156 @@ def reset_db(self):
...
 
-        raw_users_query = """CREATE TABLE USERS (
-                             U_ID INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
-                             U_NAME VARCHAR(32),
-                             U_AGE INT,
-                             U_HOME_ADDR VARCHAR(128),
-                             U_PHONE_NUMBER VARCHAR(32) )"""
-        cursor.execute(raw_users_query)
-        print(('Created new {0} table in {1}').format('USERS',self.db))
+        create_customer_table = """CREATE TABLE CUSTOMER (
```
Comment (maknoon) [inline airplanedb.py]: can you fix the whitespace but converting all your tabs to spaces (1 tab = 4 space alignment) thanks

**Review by maknoon — CHANGES_REQUESTED** [review]

**Comment by maknoon** [comment]
other than spacing issue this PR looks good!

File: airplanedb.py (hunk 1 above)
Comment (jefftwin) [reply airplanedb.py]: Did this fix the spacing issue?

Comment (maknoon) [reply airplanedb.py]: looks like it, thanks! & good job on fixing the cursor bug too

**Review by maknoon — APPROVED** [review]

**Pull request merged**